.travis.yml

```diff
@@ -62,12 +65,14 @@ before_script:
         echo "Docs build finished with the exit code ($res)."
 
         if [ $res -gt 0 ]; then
-            echo "Not attempting to run doctr due to the docs failed to build"
+            echo "Not attempting to run doctr."
         else
             pip install doctr
```

Comment:
This install step can go inside the condition too.

Reply:
Done